genetic_algorithms/problems/magic_square/benchmarks/ms_1

```diff
@@ -0,0 +1,2 @@
+15
+2 6 4 9 3 1 8 5 7
```

Comment:
Dlaczego postanowiłeś reprezentować problem jako wiersz liczb. 
Czy nie lepiej by było zrobić to macierzą któa od razu pokazywała by cały kwadrat?